website/source/docs/concepts/tokens.html.md

```diff
@@ -42,14 +42,53 @@ Finally, the `-policy` flag can be used to set the policies associated
 with the token. Learn more about policies on the
 [policies concepts](/docs/concepts/policies.html) page.
 
+<a name="ttls-and-leases" />
 ## Token Time-To-Live and Leases
 
-Every non-root token has a time-to-live (TTL) associated with it.  After the
-TTL is up, the token will no longer function. In addition to no longer
-functioning, Vault will revoke it.
+Every non-root token has a time-to-live (TTL) associated with it, which is
+the number of seconds the token is valid for, _measured from the moment it is
+created_. After the TTL is up, the token will no longer function, and
+Vault will revoke it. You can find how much of a token's TTL is remaining
+by calling `vault token-lookup`.
 
 When a token is revoked, any leases associated with the token will be revoked
 as well, even if the TTLs on the individual leases are not yet up. For example,
 if a user requests AWS access keys, after the token expires the AWS access keys
-will also be revoked. In order to avoid your token being revoked, the `vault
-token-renew` command should be periodically used to renew the token.
+will also be revoked.
+
+To prevent your token being revoked, you should call the `vault
+token-renew` command periodically, which will attempt to extend the token's TTL.
+You can specify _the new TTL_ using the `increment` argument. If this is
+not specified, then Vault will use the default TTL specified by the
+`auth/token` mount, or the global [`default_lease_ttl`](/docs/config/index.html#default_lease_ttl).
```

Comment:
But may use something else. For instance, this isn't a parameter in token roles right now, but could be at some point.